pkg/ddevapp/global_dotddev_assets/commands/host/dbeaver

```diff
@@ -7,9 +7,16 @@
 ## OSTypes: darwin,linux,wsl2
 ## HostBinaryExists: /mnt/c/Program Files/dbeaver/dbeaver.exe,/Applications/DBeaver.app,/usr/bin/dbeaver,/usr/bin/dbeaver-ce,/usr/bin/dbeaver-le,/usr/bin/dbeaver-ue,/usr/bin/dbeaver-ee,/var/lib/flatpak/exports/bin/io.dbeaver.DBeaverCommunity,/snap/bin/dbeaver-ce
 
-if [ "${DDEV_PROJECT_STATUS}" != "running" ]; then
+if [ "${DDEV_PROJECT_STATUS}" != "running" ] && [ -z "$no_recursion" ]; then
```

Comment:
Simple and tricky technique!

Reply:
Do we have a test that validates that env variables passed in like this to ddev are passed through to custom commands? If not, this seems like something that could regress easily if we change anything about how the DDEV env vars are passed in to custom commands.

Reply:
They have to be, or all the environment variables we support would not work. 

For example, `DDEV_DEBUG=true ddev launch` will demonstrate this, and TestLaunchCommand depends on this.

https://github.com/ddev/ddev/blob/6945dd019d6f378721210dbc1b83f7c8a70c4cdd/cmd/ddev/cmd/commands_test.go#L301